### ISSUE TITLE
Weird "Shift+Arrow Down" behavior in `st` terminal after updating tmux to 2.6.0

### ISSUE THREAD

**Issue by actionless** [tmux/tmux#1105]
Steps to reproduce:

1) `st -e tmux`
2) press `Shift` + `Arrow Up`
3) press `Shift` + `Arrow Down`
4) observe the codes

Expected result:
Shift+Arrow Up is `[1;2A`
Shift+Arrow Down is `[1;2B`

Actual result:
Both Shift+Arrow Up or Down are `[1;2A`

NOTE1
Bug is not reproducing when running just `st` and next running `tmux` command inside.
Also i've compared `$TERM` env var, it's the same in both cases.

NOTE2
Bug is not reproducing with tmux version 2.5.3.

**Comment by nicm** [comment]
Logs please.

**Comment by nicm** [comment]
Never mind, found the problem. You can either make this change:

~~~~
--- tty-keys.c  29 Aug 2017 09:18:48 -0000      1.101
+++ tty-keys.c  9 Oct 2017 11:33:16 -0000
@@ -271,7 +271,7 @@ static const struct tty_default_key_code
        { TTYC_KDC5, KEYC_DC|KEYC_CTRL|KEYC_XTERM },
        { TTYC_KDC6, KEYC_DC|KEYC_SHIFT|KEYC_CTRL|KEYC_XTERM },
        { TTYC_KDC7, KEYC_DC|KEYC_ESCAPE|KEYC_CTRL|KEYC_XTERM },
-       { TTYC_KIND, KEYC_UP|KEYC_SHIFT|KEYC_XTERM },
+       { TTYC_KIND, KEYC_DOWN|KEYC_SHIFT|KEYC_XTERM },
        { TTYC_KDN2, KEYC_DOWN|KEYC_SHIFT|KEYC_XTERM },
        { TTYC_KDN3, KEYC_DOWN|KEYC_ESCAPE|KEYC_XTERM },
        { TTYC_KDN4, KEYC_DOWN|KEYC_SHIFT|KEYC_ESCAPE|KEYC_XTERM },
~~~~
Or try this in .tmux.conf:
~~~~
set -as terminal-overrides ',st*:kind@:kri@'

**Comment by actionless** [comment]
Thanks!
`set -as terminal-overrides ',st*:kind@:kri@'` indeed helped

**Comment by nicm** [comment]
Great thanks, will be fixed in 2.7 anyway.
